content/applications/finance/accounting/vendor_bills.rst

```diff
@@ -75,7 +74,20 @@ Bill Payment
 Upon payment of the vendor bill, click on :guilabel:`Register Payment`. A new window pops up.
```

Comment:
```suggestion
Upon payment of the vendor bill, click on :guilabel:`Register Payment` to open a new payment window.
```
I know this wasn't your sentence originally, but it sounds awkward so let's change it :)